config/mock-api-data.js

```diff
@@ -11,6 +11,26 @@ function * range (max)
     return max;
 }
 
+function setTagName () {
+    
+    const tagsArray = [
+        'florist',
+        'grocery',
+        'pharmacy',
+        'newsstand',
+        'butcher',
+        'bakery',
+        'cafe',
+        'coffeshop',
+        'clothes',
+        'furnishings'
+    ];
+
+    const randomNumber = () => Math.floor(Math.random() * tagsArray.length); 
```

Comment:
Faker should be used instead -> it's added anyway, and provides much cleaner API.
See [Faker.js documentation](https://github.com/Marak/faker.js/wiki/Basic-Random-Data) for further reference.

Reply:
Fixed in c6cfe93